client/js/screens/FindLocker/FindLocker.js

```diff
@@ -1,10 +1,28 @@
-import React from "react";
-import { Text, View } from "react-native";
+import React, { Component } from "react";
+import MapViewComponent from "../../components/MapView";
+import { View } from "react-native";
+import GoogleMapsSearch from "../../components/GoogleMapSearch";
+// import { SearchContext } from "../../components/GoogleMapSearch";
+import GooglePlaceContext from "../../context/GooglePlaceContext";
+import styles from "./styles";
 
-const FindLocker = ({ params }) => (
-  <View>
-    <Text>This is the Find Locker Screen</Text>
-  </View>
-);
+class FindLocker extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Please remove this constructor if you do not have a state for this component and this component should be a stateless component.